src/components/pages/Onboarding/Onboarding.tsx

```diff
@@ -81,6 +81,10 @@ export class OnboardingView extends React.Component<
    * the associated video download url into state.
    */
   initializeDeeplinkingParams = async () => {
+    if (this.state.deeplinkInitialized) {
```

Comment:
I'm not clear on this, but is this a good place to use a setState callback since seems the goal is to call `setState` with `deeplinkInitialized: true` exactly once? https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous

Reply:
You're right, I moved it out of state!